package/gargoyle/files/www/js/access.js

```diff
@@ -591,40 +576,43 @@ function updateVisibility()
+	document.getElementById("noip6redi").style.display = remoteWebProtocol == "disabled" || (remoteWebProtocol == "https" && localWebProtocol == "redirect") ? "none" : "";
 
 	// RFC 1918 filter checkbox.
 	var disableRfc1918Filter = document.getElementById("disable_rfc1918_filter").checked;
 	// Either local or remote web protocol.
-	var accessWebProtocol = getSelectedValue(accessArea + "_web_protocol");
+	var accessWebProtocol = getSelectedValue(accessSide + "_web_protocol");
 	// Warn when about to lock out by disabling remote web access.
-	var protocolLockout = accessArea == "remote" && remoteWebProtocol == "disabled";
+	var protocolLockout = remoteView && remoteWebProtocol == "disabled";
 	// Warn when about to lock out by enabling RFC 1918 filter again.
-	var rfc1918FilterLockout = accessArea == "remote" && remoteWebProtocol != "disabled" && !disableRfc1918Filter && ipInPrivate(REMOTE_ADDR) && !ipInPrivate(SERVER_ADDR);
+	var rfc1918FilterLockout = remoteView && remoteWebProtocol != "disabled" && !disableRfc1918Filter &&
+		isIPv4(REMOTE_ADDR) && ipInPrivate(REMOTE_ADDR) && isIPv4(SERVER_ADDR) && !ipInPrivate(SERVER_ADDR);
 	// Warn when about to lock out on unfit protocol change if uhttpd is used as a reverse proxy's backend.
 	var reverseProxyLockout = !directView && accessWebProtocol != "disabled" && accessWebProtocol != "both" && accessWebProtocol.replace("redirect", "https") != serverProt;
 	// Reset warning container for either local or remote web protocol.
-	var accessWebProtocolLockoutContainer = document.getElementById(accessArea + "_web_protocol_lockout_container");
+	var accessWebProtocolLockoutContainer = document.getElementById(accessSide + "_web_protocol_lockout_container");
 	accessWebProtocolLockoutContainer.style.display = "none";
 	accessWebProtocolLockoutContainer.innerHTML = "";
 	// Reset warning container for RFC 1918 filter checkbox.
 	var rfc1918FilterLockoutContainer = document.getElementById("rfc1918_filter_lockout_container");
 	rfc1918FilterLockoutContainer.style.display = "none";
 	rfc1918FilterLockoutContainer.innerHTML = "";
 	var lockoutTemplate =
-	'<span class="alert alert-danger col-xs-12" role="alert" id="lockout">' +
-		'<p>' +  accessStr.LockoutWarning + '</p>' +
-		'<span id="reverse_proxy_smooth" style="display: none">' +
-			'<p>' + accessStr.ReverseProxySmooth + ':</p>' +
-			'<ol>' +
-				'<li>' + accessStr.ReverseProxySelect + ' <i>HTTP & HTTPS</i> ' + accessStr.ReverseProxySaveCh + '.</li>' +
-				'<li>' + accessStr.ReverseProxySwitch + ' <i><span id="reverse_proxy_switch"></span></i>.</li>' +
-				'<li>' + accessStr.ReverseProxySelect + ' <i><span id="reverse_proxy_select"></span></i> ' + accessStr.ReverseProxySaveCh + '.</li>' +
-			'</ol>' +
-		'</span>' +
-		'<p>' + accessStr.AccessChain + ':</p>' +
-		'<p><span id="access_chain"></span></p>' +
-	'</span>';
+	`<span id="lockout" class="col-xs-12">
```

Comment:
Please don't use backtick templating. This does not seem to play nicely with ./i18n-scripts/localize.py.
Alternatively, please fix localize.py to correctly address backtick template variables.

Reply:
Removed all backtick templates.

---
I haven't fully understood `localize.py` but it seems to work fine with backtick templates. It replaces `${jsObj.Variable}` with `${"Localized Text"}` which is totally valid and respects original escaping. The syntax is `${ expression }` where expression is not limited to variable names.

Reply:
But it can be problematic as soon as there is a backtick within the localized text.